### PR TITLE
chore(demo): decompose embed page

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes # 

## What is the new behavior?

image

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/demo/src/modules/components/editor/embed/youtube/editor-embed-youtube.component.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<tui-doc-page
+    header="Editor - Youtube"
+    package="ADDON-EDITOR"
+    type="components"
+>
+    <ng-template pageTab>
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/editor/embed/youtube/editor-embed-youtube.component.html:6]: Do we actually need to create tab if it is the single tab on the page ? 🤔 

Screenshot 2023-02-08 at 13 03 44

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/app/pages.ts [hunk 2]
```diff
@@ -952,9 +952,27 @@ export const pages: TuiDocPages = [
...
+            {
+                section: $localize`Editor`,
+                title: `Iframe`,
+                keywords: `editor, embed, iframe, wysiwyg, редактор, айфрейм, html, rich, text`,
+                route: `/editor/embed/iframe`,
+            },
+            {
+                section: $localize`Editor`,
+                title: `HTML5`,
+                keywords: `editor, embed,  video player, audio player, html5, wysiwyg, редактор, плеер, html, rich, text`,
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/demo/src/modules/app/pages.ts]: fixed 
image

File: projects/demo/src/modules/components/editor/embed/youtube/editor-embed-youtube.component.html (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/components/editor/embed/youtube/editor-embed-youtube.component.html]: image

File: projects/addon-doc/src/components/page/page.style.less [hunk 3]
```diff
@@ -28,6 +25,14 @@
     }
 }
 
+.t-title-wrapper {
```
Comment (splincode) [inline projects/addon-doc/src/components/page/page.style.less:28]: Before:


image

After:
image
image

File: projects/addon-doc/src/components/page/page.template.html [hunk 4]
```diff
@@ -44,13 +55,6 @@ <h1 class="t-title">
             </ng-template>
         </ng-container>
     </tui-tabs-with-more>
-    <tui-doc-source-code
-        class="t-source-code"
-        [header]="header"
-        [package]="package"
-        [type]="type"
-        [path]="path"
-    ></tui-doc-source-code>
```
Comment (nsbarsukov) [inline projects/addon-doc/src/components/page/page.template.html:53]: diff

Comment (splincode) [reply projects/addon-doc/src/components/page/page.template.html]: Most likely it is better to discuss with UI designer in a separate issue. I revert back style changes

**Review by waterplea — APPROVED** [review]

**Pull request merged**